docs/user/discover.asciidoc

```diff
@@ -3,101 +3,210 @@
-you can specify a common or recently-used time range, a relative time
-from now, or an absolute time range.
+For this tutorial, your goal is to better manage your product inventory.  You want to
+know the top-selling products and on what day of the week these products sell the most.
 
 [float]
-=== Search your data
+[[find-the-data-you-want-to-use]]
+=== Step 2. Find your data
+
+Tell {kib} where to find the data you want to explore, then specify the time range in which to view that data.
+
+. Open the main menu, and select **Discover**.
+
+. Select the data you want to work with.
++
+{kib} uses an <<index-patterns,index pattern>> to tell it where to find
+your {es} data.
+To view the ecommerce sample data, make sure the index pattern is set to **kibana_sample_data_ecommerce**.
++
+[role="screenshot"]
+image::images/discover-index-pattern.png[How to set the index pattern in Discover]
+
+. Adjust the time range to view data for the *Last 7 days*.
++
+NOTE: The range selection is based on the default time field in your data.
+If you are using the sample data, this value was set when you added the data.
+If you  are using your own data, and it does not have a time field, the range selection is not available.
 
-Now that you have your data and set the time span, you can start asking your questions.
-You can search your data using the <<kuery-query, Kibana Query language>>,
-which offers a simplified query syntax.
-For example, if
-you search for `day_of_week : Friday`, you'll get a list of all documents
-in which `day_of_week` is set to `Friday`. If you prefer
-<<lucene-query, Lucene query syntax>>, you can access it from the KQL menu.
+. To view the count of documents for a given time in the specified range,
+click and drag the mouse over the histogram, and then click *Refresh*.
 
 [float]
-=== Filter your search results
+[[explore-fields-in-your-data]]
+=== Step 3. Explore the fields in your data
 
-Next, you'll want narrow your search results to a more manageable data set.
-When you click on a name in the field list, you'll see
-the top five values for the field, the number of documents that contain the field,
-and the percentage of documents that contain each value. From this view, you can
-use the (+) magnifier icon to quickly find all
-documents that have that value, or (-) to exclude all
-documents with that value. For more filter options, see <<field-filter, filtering by field>>.
+**Discover** includes a table that shows all the documents that match your search.
+By default, the table includes columns for the time field and the document `_source`,
+which can be overwhelming. You’ll modify this table to display only your fields of interest.
 
+. Scan through the list of **Available fields** to see
+what’s in your data. You can also search for a field by name.
+
+. Find the `manufacturer` field, and then click it to view the five most popular values for that field.
```

Comment:
Just a side note, not sure if we should mention it. Discover fetches a maximum of 500 docs, and the popular values are calculated by these 500 docs